app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
```

Comment:
- According to the project requirements, your `application.rb` file should have a `current_user` method that selects the first user. Please fix this in the right file 👍. Doing this will ensure that the `current_user` method is accessible to all controllers, hence avoiding repition 😄